src/protocol.errorcode.ts

```diff
@@ -13,6 +13,7 @@ export enum ProtocolErrorCode {
     VALIDATION_EXCEPTION = 'ValidationException',
     DUPLICATE_ENTRY = 'DuplicateEntry',
     NO_CITIZEN_FOUND = 'NoCitizenFound',
+    NOT_FOUND = 'NotFound',
```

Comment:
since your changing this 'enum' what do you think of adding a not implemented value?  is it worth it?

Reply:
good call, I'll add that too